### DISCUSSION TITLE
Make Suspenders fail if running with an unsupported Rails version

### DISCUSSION THREAD

**Discussion by thiagoa** [thoughtbot/suspenders#1303] [Ideas]
The suspenders command does not fail when accidentally picking up an unsupported Rails version, which may lead to very strange errors. I had a problem with it picking up Rails 6.1 (actually, Rails 6.1 was uninstalled but it picked up railties 6.1.4.1, which was still installed), so to fix it I installed Rails 6.0 and ran suspenders with the following command:

```sh
suspenders _6.0.4.1_ my-app
```

**Comment by stevepolitodesign** [comment]
Although we no longer have a system executable, we should still account for this in the [application template](https://github.com/thoughtbot/suspenders/blob/main/lib/install/web.rb). We already do this for Node version:

https://github.com/thoughtbot/suspenders/blob/efe875c0810902766b836acc93f9a80b97f685dc/lib/install/web.rb#L14-L24
